src/status_im/contexts/centralized_metrics/tracking.cljs

```diff
@@ -62,4 +78,15 @@
     :set-view-id
     (track-view-id-event second-parameter)
 
+    :wallet/select-account-tab
+    (when (= second-parameter :collectibles)
+      (navigated-to-collectibles-tab-event :account))
+
+    :wallet/select-home-tab
+    (when (= second-parameter :collectibles)
+      (navigated-to-collectibles-tab-event :home))
+
+    :wallet/flush-collectibles-fetched
+    (collectilbes-fetched-event db)
```

Comment:
Tracking this parameter was solely my initiative and as a developer I felt comfortable to track as much info as possible :) But agree that if we intend to track less data then this is a wrong approach.
But I think it would be valuable to know if user has any collectibles or not. We can pass a boolean parameter to event, like `:has-collectibles?`. @shivekkhurana, @ilmotta, wdyt?

Reply:
I agree with you @vkjr, knowing if users have collectibles is useful and it's a good middle ground in terms of privacy because knowing exactly how many collectibles users have seem not super necessary.

Reply:
The number of collectibles tell us if people are using the NFT features. If there are no collectibles for the majority of the users, then this screen becomes less important.